<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Авторизация Вконтакте</string>
  </property>
  <widget class="QPushButton" name="auth_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Авторизоваться</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>461</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин VK:</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>461</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль VK:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>461</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="tFA_checkbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Двухфакторная аутентификация</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
